openmdao/utils/om.py

```diff
@@ -60,7 +60,10 @@ def _n2_setup_parser(parser):
     parser : argparse subparser
         The parser we're adding options to.
     """
-    parser.add_argument('file', nargs=1, help='Python script or recording containing the model.')
+    parser.add_argument('file', nargs=1,
+                        help='Python script or recording containing the model. '
+                        'For separate recording metadata from a parallel run, specify both '
```

Comment:
I think this phrasing is a bit unclear... suggest:
"If metadata was recorded in a separate file (e.g. for a parallel run), specify both "

Reply:
Updated